### ISSUE TITLE
"Emulate idle detection states" exists in Edge?

### ISSUE THREAD

**Issue by pankajparashar** [captainbrosset/devtools-tips#55]
The Sensory option to emulate Idle states also exists in Edge right?

<img width="952" alt="image" src="https://user-images.githubusercontent.com/38640616/210866031-c4105761-8e8d-450c-82f2-1eefd8c93317.png">

**Comment by captainbrosset** [comment]
Right, what confuses me is that the IdleDetection API doesn't seem to be supported in Edge: https://developer.mozilla.org/en-US/docs/Web/API/IdleDetector#browser_compatibility
Maybe this is a problem on MDN/BCD data. Or maybe Edge just disabled the API downstream. We'll need to find out.
